wire-ios-mono.xcworkspace/xcshareddata/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we want to add these changes?

Reply:
Yes, I'd like to keep it until an issue is fixed.
It breaks into the debugger whenever a viewController's present method is called while the controller is already presenting.